adapters/denyChecker/builder_test.go

```diff
@@ -36,15 +36,13 @@ func TestAll(t *testing.T) {
 	}
 	listChecker := a.(adapters.ListChecker)
 
-	var ok bool
-	ok, err = listChecker.CheckList("")
-	if ok || err != nil {
-		t.Error("Expecting to always get false")
-	}
-
-	ok, err = listChecker.CheckList("ABC")
-	if ok || err != nil {
-		t.Error("Expecting to always get false")
+	cases := []string{"", "ABC", "ABC/DEF", "AAA", "123"}
+	for _, c := range cases {
+		var ok bool
+		ok, err = listChecker.CheckList(c)
+		if ok || err != nil {
+			t.Error("Expecting to always get false, got success with " + c)
```

Comment:
While we are here, can we make the error message a bit more informative?

Something like:

`t.Errorf("CheckList(%s) => got 'true'; want 'false'", c)`

Same comment for other locations in the tests.